content/en/docs/pipelines/installation/overview.md

```diff
@@ -97,11 +112,17 @@ Upgrade Support (**Alpha**)
 : Reinstallation supported with [instructions](/docs/gke/pipelines/upgrade). Upgrade support is limited.
 
 Notes on specific features
-: After deployment, your Kubernetes cluster includes all the 
+:
+  * After deployment, your Kubernetes cluster includes all the 
   [Kubeflow components](/docs/components/).
   For example, you can use the Jupyter notebook services 
   [deployed with Kubeflow](/docs/notebooks/) to create one or more notebook 
   servers in your Kubeflow cluster.
+  * KFP multi user support is **only available** in full Kubeflow. It supports
```

Comment:
I suggest to use "Kubeflow Pipelines (KFP)" once in the doc and then refer to it as "KFP" or stick to "Kubeflow Pipelines" everywhere, like you did before.

> "* KFP multi user support is **only available** in full Kubeflow..."
Can you check if the multi-user feature is in v1.1 since you're saying it's in "full Kubeflow" (users may not understand what "full Kubeflow" is and you've mentioned this feature is in v1.1?)

And: "multi-user" with a dash.